tasks/main.yml

```
@@ -2,6 +2,27 @@
 - name: Setup firewalld
   include_tasks: firewalld.yml
 
+- name: Collect Service Facts
+  service_facts:
+  when: firewall_disable_conflicting_services
```

```suggestion
  when: firewall_disable_conflicting_services | bool
```
see https://github.com/redhat-cop/automation-good-practices/tree/main/coding_style#ansible-guidelines "Use the | bool filter when using bare variables"